source/MissionAction.cpp

```diff
@@ -349,7 +364,9 @@ MissionAction MissionAction::Instantiate(map<string, string> &subs, const System
 
 	if(!conversation->IsEmpty())
 		result.conversation = ExclusiveItem<Conversation>(conversation->Instantiate(subs, jumps, payload));
-
+	
```

Comment:
Did you mess with tabs and spaces mb? Weird that it showsthe line being removed and added like this